tools/cldr/cldr-to-icu/src/test/java/org/unicode/icu/tool/cldrtoicu/AlternateLocaleDataTest.java

```diff
@@ -97,10 +97,10 @@ public void testMissingTarget() {
         CldrData unresolved = transformed.getDataForLocale("xx", UNRESOLVED);
         CldrData resolved = transformed.getDataForLocale("xx", RESOLVED);
 
-        // Even though the missing target is not matched (so no change there) the source is always
-        // removed from the transformed data.
-        assertValuesUnordered(unresolved, other);
-        assertValuesUnordered(resolved, other);
+        // If there's no target the alt-path mapping is incomplete and we do nothing (this matches
+        // the old CLDR tool behaviour and reasonable but can hide inconsistencies in CLDR data).
+        assertValuesUnordered(unresolved, source, other);
```

Comment:
This was technically unrelated but broke in the previous merge (which effectively undid some of the alt-path fixes I made earlier). The code itself is (and was) correct, it's just this one test that got muddled up.
This reverts the comment to previous (we no longer expect a change) and gets the expectations correct by including the original source.